src/components/files/deleteFiles.ts

```diff
@@ -24,6 +24,11 @@ export const deleteFile = async (file: FileDetail) => {
     if (!thumbnailPath.match(genericThumbnailsFolderName))
       await fsPromises.unlink(path.join(filesPath, thumbnailPath))
     console.log(`File deleted: ${originalFilename}`)
+
+    // Also delete folder if it's now empty
+    const dir = path.dirname(filePath)
+    if (fs.readdirSync(path.join(filesPath, dir)).length === 0)
```

Comment:
I think apart from snapshot things, we should used fsPromise and await

Reply:
Okay, I changed to this: dc696156

So this should be exactly the same functionality?
```ts
    const dir = path.dirname(filePath)
    if ((await fsPromises.readdir(path.join(filesPath, dir))).length === 0)
      await fsPromises.rmdir(path.join(filesPath, dir))
  } catch (err) {
    console.log(err)
```

Reply:
yep, thanks